node/process.ts

```diff
@@ -231,6 +232,27 @@ memoryUsage.rss = function (): number {
   return memoryUsage().rss;
 };
 
+export function kill(pid: number, sig: Deno.Signal | number = "SIGTERM") {
+  if (pid != (pid | 0)) {
+    throw new ERR_INVALID_ARG_TYPE("pid", "number", pid);
+  }
+
+  if (typeof sig === "string") {
+    try {
+      Deno.kill(pid, sig);
+    } catch (e) {
+      if (e instanceof TypeError) {
+        throw new ERR_UNKNOWN_SIGNAL(sig);
+      }
+      throw e;
+    }
+  } else {
+    throw new ERR_UNKNOWN_SIGNAL(sig.toString());
```

Comment:
nit: It might be better to make this `notImplemented()` error (especially when the signal is a number)